tests/rptest/tests/schema_registry_test.py

```diff
@@ -1351,13 +1351,19 @@ def bool_alpha(b: bool) -> str:
     @cluster(num_nodes=3)
     def test_restarts(self):
         admin = Admin(self.redpanda)
+        largest_et = -1.0
 
         def check_connection(hostname: str):
-            result_raw = self._get_subjects(hostname=hostname)
+            start_t = time.time()
+            result_raw = self._get_subjects(hostname=hostname, timeout=None)
+            end_t = time.time()
```

Comment:
I am measuring the execution time so we can grasp a new timeout value for the REST request. I will update this after a few test runs in CI.